azure-devops/azext_devops/dev/boards/area.py

```diff
@@ -0,0 +1,172 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.util import CLIError
+from azext_devops.devops_sdk.v5_0.work_item_tracking.models import WorkItemClassificationNode
+from azext_devops.devops_sdk.v5_0.work.models import (TeamContext,
+                                                      TeamFieldValuesPatch,
+                                                      TeamFieldValue)
+from azext_devops.dev.common.arguments import convert_date_string_to_iso8601
+from azext_devops.dev.common.services import (resolve_instance_and_project,
+                                              get_work_item_tracking_client,
+                                              get_work_client)
+
+
+def get_project_areas(depth=1, path=None, organization=None, project=None, detect=None):
+    """List areas for a project.
+    :param depth: Depth of children to fetch.
+    :type depth: int
+    """
+    organization, project = resolve_instance_and_project(detect=detect,
+                                                         organization=organization,
+                                                         project=project)
+    client = get_work_item_tracking_client(organization)
+    list_of_areas = client.get_classification_node(project=project,
+                                                        structure_group='areas',
```

Comment:
Might want to put `areas` into a const with meaningful name, will be easy to understand it